### ISSUE TITLE
flickering page when number of transactions fluctuates

### ISSUE THREAD

**Issue by MonsieurNicolas** [stellar/dashboard#32]
The section "Recent operations: Live network" changes size based on the number of transactions, this causes the page to jump up/down based on the number of transactions in the previous ledger.

I would recommend picking a size (let's say 20 transactions) and have a fixed area for transactions (displaying empty space in that area if there are not enough transactions).

**Comment by bartekn** [comment]
It's not about the number of transactions (we always display last 20). It's happening because summary of an operation sometimes takes 2 lines (instead of 1):

![recent](https://user-images.githubusercontent.com/464938/36898535-88a15f6c-1e1b-11e8-820a-8116a34be4fe.png)

We have another task to update the list of recent operation every 10-15 seconds instead of real-time as now. The flickering will still be there but won't be happening that often. We may also shorten long summaries.

**Comment by MonsieurNicolas** [comment]
Yeah slowing the refresh rate doesn't seem to be the right fix... if anything I prefer it to move more. That said this is a policy choice.

All we need is to get a fixed width & height for that area, then the policy on what to display can be adjusted over time.... If we need the extra data, maybe a more horizontal format with scrolling may be more appropriate in the future
